src/main/java/com/google/api/codegen/gapic/MainGapicProviderFactory.java

```diff
@@ -282,9 +286,7 @@
     }
   }
 
-  /**
-   * Create the GapicProviders based on the given id
-   */
+  /** Create the GapicProviders based on the given id */
```

Comment:
This is actually what I have set -- I'm not sure why it's happening. I've actually just completely disabled auto-reformatting to avoid this sort of thing in the future.